<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="TimeoutException.xml" source-language="en-US" target-language="ru-RU">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac53391efb8377fbf46be71dcc8bfc02af4e0272a1b.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">3391efb8377fbf46be71dcc8bfc02af4e0272a1b</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-1.0,netstandard-1.1,netstandard-1.2,netstandard-1.3,netstandard-1.4,netstandard-1.5,netstandard-1.6,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.TimeoutException">
          <source>The exception that is thrown when the time allotted for a process or operation has expired.</source>
          <target state="translated">Исключение, возникающее при истечении времени, предоставленного процессу или операции.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.TimeoutException">
          <source>For information about setting the time-out interval before a <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph> exception is thrown, consult the documentation for the type whose method timed out. For information about handling the time-out, see the documentation for the method that threw the exception.</source>
          <target state="translated">Дополнительные сведения об установке интервала времени ожидания до <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph> исключения, обратитесь к документации для типа, метод которого истекло время ожидания. Сведения об обработке тайм-аута см. в документации для метода, вызвавшего исключение.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.TimeoutException">
          <source><ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph> uses the HRESULT, COR_E_TIMEOUT, which has the value 0x80131505.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph> использует значение HRESULT, COR_E_TIMEOUT, который имеет значение 0x80131505.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.TimeoutException">
          <source>For a list of initial property values for an instance of  <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph>, see the <ph id="ph2">&lt;xref:System.TimeoutException.%23ctor%2A&gt;</ph> constructors.</source>
          <target state="translated">Список начальных значений свойств для экземпляра <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph>, в разделе <ph id="ph2">&lt;xref:System.TimeoutException.%23ctor%2A&gt;</ph> конструкторы.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.TimeoutException">
          <source>The following code example demonstrates the use of <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph> in conjunction with members of the <ph id="ph2">&lt;xref:System.IO.Ports.SerialPort?displayProperty=nameWithType&gt;</ph> class.</source>
          <target state="translated">В следующем примере кода показано использование <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph> совместно с членами <ph id="ph2">&lt;xref:System.IO.Ports.SerialPort?displayProperty=nameWithType&gt;</ph> класса.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="T:System.TimeoutException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph> class.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor">
          <source>This constructor initializes the inherited <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> property of the new instance with a system-supplied error message.</source>
          <target state="translated">Этот конструктор инициализирует наследуемого <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> свойства нового экземпляра, используя системное сообщение.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">Это сообщение учитывает текущую культуру системы.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph>.</source>
          <target state="translated">В следующей таблице представлены исходные значения свойств экземпляра класса <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor">
          <source>Property</source>
          <target state="translated">Свойство.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor">
          <source>Value</source>
          <target state="translated">Значение</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor">
          <source>A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">Пустая ссылка (<ph id="ph1">`Nothing`</ph> в Visual Basic).</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor">
          <source>The localized error message string.</source>
          <target state="translated">Строка локализованного сообщения об ошибке.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.String)">
          <source>The message that describes the error.</source>
          <target state="translated">Сообщение, описывающее ошибку.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph> class with the specified error message.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph> указанным сообщением об ошибке.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String)">
          <source>This constructor initializes the inherited <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> property of the new instance using the value of the <ph id="ph2">`message`</ph> parameter.</source>
          <target state="translated">Этот конструктор инициализирует наследуемого <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> свойств нового экземпляра, используя значение <ph id="ph2">`message`</ph> параметра.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph>.</source>
          <target state="translated">В следующей таблице представлены исходные значения свойств экземпляра класса <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String)">
          <source>Property</source>
          <target state="translated">Свойство.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String)">
          <source>Value</source>
          <target state="translated">Значение</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String)">
          <source>A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">Пустая ссылка (<ph id="ph1">`Nothing`</ph> в Visual Basic).</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String)">
          <source>The error message string.</source>
          <target state="translated">Строка сообщения об ошибке.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object that contains serialized object data about the exception being thrown.</source>
          <target state="translated">Объект <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>, в котором содержатся сериализованные данные объекта, относящиеся к выбрасываемому исключению.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> object that contains contextual information about the source or destination.</source>
          <target state="translated">Объект <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph>, содержащий контекстные сведения об источнике или назначении.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <bpt id="p1">&lt;c&gt;</bpt>context<ept id="p1">&lt;/c&gt;</ept> parameter is reserved for future use, and can be specified as <ph id="ph1">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated">Параметр <bpt id="p1">&lt;c&gt;</bpt>context<ept id="p1">&lt;/c&gt;</ept> зарезервирован для использования в будущем, и ему можно присвоить значение <ph id="ph1">&lt;see langword="null" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">Инициализирует новый экземпляр класса <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph> с сериализованными данными.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">Данный конструктор вызывается в процессе десериализации для воссоздания объекта исключения, переданного в потоке.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>Binary Serialization<ept id="p1">](~/docs/standard/serialization/binary-serialization.md)</ept>.</source>
          <target state="translated">Дополнительные сведения см. в разделе <bpt id="p1">[</bpt>двоичной сериализации<ept id="p1">](~/docs/standard/serialization/binary-serialization.md)</ept>.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;paramref name="info" /&gt;</ph> parameter is <ph id="ph2">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated">Параметр <ph id="ph1">&lt;paramref name="info" /&gt;</ph> имеет значение <ph id="ph2">&lt;see langword="null" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The class name is <ph id="ph1">&lt;see langword="null" /&gt;</ph>, or <ph id="ph2">&lt;see cref="P:System.Exception.HResult" /&gt;</ph> is zero (0).</source>
          <target state="translated">Имя класса — <ph id="ph1">&lt;see langword="null" /&gt;</ph> или значение свойства <ph id="ph2">&lt;see cref="P:System.Exception.HResult" /&gt;</ph> равно нулю (0).</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>The message that describes the error.</source>
          <target state="translated">Сообщение, описывающее ошибку.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Исключение, которое является причиной текущего исключения.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Если значение параметра <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> не равно <ph id="ph1">&lt;see langword="null" /&gt;</ph>, текущее исключение возникает в блоке <ph id="ph2">&lt;see langword="catch" /&gt;</ph>, обрабатывающем внутреннее исключение.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph> class with the specified error message and inner exception.</source>
          <target state="translated">Выполняет инициализацию нового экземпляра класса <ph id="ph1">&lt;see cref="T:System.TimeoutException" /&gt;</ph> с указанным сообщением об ошибке и внутренним исключением.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>This constructor initializes the inherited <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> property of the new instance using the value of the <ph id="ph2">`message`</ph> parameter.</source>
          <target state="translated">Этот конструктор инициализирует наследуемого <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> свойств нового экземпляра, используя значение <ph id="ph2">`message`</ph> параметра.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Исключение, созданное как прямой результат предыдущего исключения, должно содержать в свойстве <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> ссылку на предыдущее исключение.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed to the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> Свойство возвращает то же значение, которое передается конструктору, или <ph id="ph2">`null`</ph> Если <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> свойства не предоставляет конструктору значение внутреннего исключения.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph>.</source>
          <target state="translated">В следующей таблице представлены исходные значения свойств экземпляра класса <ph id="ph1">&lt;xref:System.TimeoutException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Свойство.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Значение</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>The error message string.</source>
          <target state="translated">Строка сообщения об ошибке.</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.TimeoutException.#ctor(System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">Ссылка на внутреннее исключение.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>